src/Umbraco.Web/HealthCheck/NotificationMethods/EmailNotificationMethod.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Net.Mail;
+using System.Threading.Tasks;
+using Umbraco.Core.Configuration.HealthChecks;
+
+namespace Umbraco.Web.HealthCheck.NotificationMethods
+{
+    [HealthCheckNotificationMethod("email")]
+    public class EmailNotificationMethod : NotificationMethodBase, IHealthCheckNotificatationMethod
+    {
+        public EmailNotificationMethod(bool enabled, bool failureOnly, HealthCheckNotificationVerbosity verbosity,
+                string recipientEmail, string subject)
```

Comment:
This subject should just be localized text too instead of being in another config section this would also mean that if people set their default culture to a different language it will just work, i'll fix